celestia/peers.txt

```diff
@@ -1,3 +1,4 @@
+87af580078c80c630625db2360b3dc19483d29cd@185.182.194.163:26656
```

Comment:
please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`